Do not show breadcrumbs for worldwide organisation and office pages

jkempster34 opened the pull request:
https://trello.com/c/MIgcASTq

### Refactor how we define which pages show default breadcrumbs
Currently, we are maintaining multiple lists of which content items should not
show breadcrumbs in the `ContentItemsController`.

This moves the logic that determines whether a page shows
breadcrumbs into the derived classes of `ContentItemPresenter`.

### Do not show breadcrumbs for worldwide organisation and office pages

We don't want to show breadcrumbs on the worldwide organisation and office
pages.

### Add worldwide office examples to the development controller

### Fix spacing above logo
We updated the class names for the styling of this page, but missed the logo
class.

### Breadcrumbs removed
|This PR|Prod|
|-|-|
|https://government-frontend-pr-2812.herokuapp.com/world/organisations/british-embassy-paris/office/british-embassy|https://www.gov.uk/world/organisations/british-embassy-paris/office/british-embassy|

### Breadcrumbs the same (history pages)
|This PR|Prod|
|-|-|
|https://government-frontend-pr-2812.herokuapp.com/government/history/10-downing-street|https://www.gov.uk/government/history/10-downing-street|
|https://government-frontend-pr-2812.herokuapp.com/government/history/11-downing-street|https://www.gov.uk/government/history/11-downing-street|
|https://government-frontend-pr-2812.herokuapp.com/government/history/1-horse-guards-road|https://www.gov.uk/government/history/1-horse-guards-road|
|https://government-frontend-pr-2812.herokuapp.com/government/history/king-charles-street|https://www.gov.uk/government/history/king-charles-street|
|https://government-frontend-pr-2812.herokuapp.com/government/history/lancaster-house|https://www.gov.uk/government/history/lancaster-house|
|https://government-frontend-pr-2812.herokuapp.com/government/history|https://www.gov.uk/government/history|

### Breadcrumbs the same (service manual pages)
||This PR|Prod|
|-|-|-|
|Topic|https://government-frontend-pr-2812.herokuapp.com/service-manual/helping-people-to-use-your-service|https://www.gov.uk/service-manual/helping-people-to-use-your-service|
|Guide|https://government-frontend-pr-2812.herokuapp.com/service-manual/design/introduction-designing-government-services|https://www.gov.uk/service-manual/design/introduction-designing-government-services|
|Homepage|https://government-frontend-pr-2812.herokuapp.com/service-manual|https://www.gov.uk/service-manual|
|Service standard|https://government-frontend-pr-2812.herokuapp.com/service-manual/service-standard|https://www.gov.uk/service-manual/service-standard|
|Toolkit|https://government-frontend-pr-2812.herokuapp.com/service-toolkit|https://www.gov.uk/service-toolkit|

---

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

Follow [these steps](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html) if you are doing a Rails upgrade.

brucebolt reviewed (approved):
The changes all make sense to me.

Probably worth getting the owning team to review this too.

chao-xian reviewed (changes requested):
Thanks for the URLs @jkempster34. I can see that the spacing above the logo isn't right. It should be like on https://webarchive.nationalarchives.gov.uk/ukgwa/20220203051007/https://www.gov.uk/world/organisations/british-embassy-paris/office/british-embassy

jkempster34 commented:
> Thanks for the URLs @jkempster34. I can see that the spacing above the logo isn't right. It should be like on https://webarchive.nationalarchives.gov.uk/ukgwa/20220203051007/https://www.gov.uk/world/organisations/british-embassy-paris/office/british-embassy

Good spot! Fixed in a separate commit.

I've also added some examples to the development controller in another commit

chao-xian reviewed (approved):
Thanks @jkempster34 !
